docs/kyma/docs/04-04-use-your-own-domain.md

```diff
@@ -104,37 +95,49 @@ Follow these steps:
 
 ### Get the TLS certificate
 
-1. Create a folder for certificates. Run:
+1. Export issuer email and domain for the cluster:
+    ```
+    export CERT_ISSUER_EMAIL={YOUR_EMAIL}
+    export DOMAIN="$CLUSTER_NAME.$(echo $DNS_NAME | sed `s/\.$//`)"
+    ```
+
+2. Create a folder for certificates. Run:
     ```
     mkdir letsencrypt
     ```
-2. Create a new service account and assign it to the `dns.admin` role. Run these commands:
+3. Create a new service account and assign it to the **dns.admin** role. Run these commands:
     ```
-    gcloud iam service-accounts create dnsmanager --display-name "dnsmanager" --project "$PROJECT"
+    gcloud iam service-accounts create dnsmanager --display-name "dnsmanager" --project "$GCP_PROJECT"
     ```
     ```
-    gcloud projects add-iam-policy-binding $PROJECT \
-        --member serviceAccount:dnsmanager@$PROJECT.iam.gserviceaccount.com --role roles/dns.admin
+    gcloud projects add-iam-policy-binding $GCP_PROJECT \
+        --member serviceAccount:dnsmanager@$GCP_PROJECT.iam.gserviceaccount.com --role roles/dns.admin
     ```
 
-3. Generate an access key for this account in the `letsencrypt` folder. Run:
+    > **NOTE**: You don't have to create a new DNS manager service account (SA) every time you deploy a cluster. Instead, you can use an existing SA that has the **dns.admin** assigned.
+
+
+4. Generate an access key for this account in the `letsencrypt` folder. Run:
     ```
-    gcloud iam service-accounts keys create ./letsencrypt/key.json --iam-account dnsmanager@$PROJECT.iam.gserviceaccount.com
+    gcloud iam service-accounts keys create ./letsencrypt/key.json --iam-account dnsmanager@$GCP_PROJECT.iam.gserviceaccount.com
     ```
-4. Run the Certbot Docker image with the `letsencrypt` folder mounted. Certbot uses the key to apply DNS challenge for the certificate request and stores the TLS certificates in that folder. Run:
+    
+    > **NOTE**: There is a fixed number of keys that may be generated for single Service Account. We suggest reusing keys instead of generating new one for every cluster.
```

Comment:
```suggestion
    > **NOTE**: The number of keys you can generate for a single service account is limited. Reuse the existing keys instead of generating a new key for every cluster.
```